api/types/types.go

```diff
@@ -169,9 +169,9 @@ type BuildCache struct {
 
 // BuildCachePruneOptions hold parameters to prune the build cache
 type BuildCachePruneOptions struct {
-	All         bool
-	KeepStorage int64
```

Comment:
We probably should keep it as `Deprecated` for at least one version.

Reply:
Bump, I think this went unaddressed :)

Reply:
Ah, I misunderstood the diff. I thought I had kept this attribute.